engines/bladerunner/bladerunner.h

```diff
@@ -316,16 +318,15 @@ class BladeRunnerEngine : public Engine {
 
 	GUI::Debugger *getDebugger();
 	Common::String getTargetName() const;
-};
 
-static inline const Graphics::PixelFormat gameDataPixelFormat() {
-	return Graphics::PixelFormat(2, 5, 5, 5, 1, 10, 5, 0, 15);
-}
+	const Graphics::PixelFormat gameDataPixelFormat() const {
+		return Graphics::PixelFormat(2, 5, 5, 5, 1, 10, 5, 0, 15);
+	}
 
-static inline const Graphics::PixelFormat screenPixelFormat() {
-	// Should be a format supported by Android port
-	return Graphics::PixelFormat(2, 5, 5, 5, 1, 11, 6, 1, 0);
-}
+	Graphics::PixelFormat screenPixelFormat() {
```

Comment:
you dont need to move this in into the class, if you keep it out and use ((BladeRunnerEngine *)g_engine) then you dont need the other change you made